src/language.js

```diff
@@ -284,6 +284,12 @@ const pages = {
     totalAbundanceLabel: 'Total Abundance',
     totalBiomassLabel: 'Total Biomass (kg/ha)',
     newBenthicAttributeLink: 'Propose New Benthic Attribute...',
+    formSectionTitle: {
+      sampleEvent: 'Sample Event',
+      quadrat: 'Quadrat Collection',
```

Comment:
Can we make the key `quadratcollection`? A small thing, but the distinction is what led to the current user confusion in the first place, so best to be consistent.

Reply:
changed.